app/admin/partners/page.tsx

```
@@ -0,0 +1,1263 @@
+import { Building, FileText, MapPin, Plus, Search, Users, Filter, Globe, Mail, Phone, Edit, Trash2, Calendar, User } from "lucide-react"
+import { Avatar, AvatarFallback, AvatarImage } from "@/components/ui/basic/avatar"
+import { extendedPalette } from "@/lib/colors"
+import { MultiPurposeModal } from "@/components/ui/overlay/multi-purpose-modal"
+import { Tabs, TabsList, TabsTrigger } from "@/components/ui/navigation/tabs"
+import { Label } from "@/components/ui/basic/label"
+import { Textarea } from "@/components/ui/form/textarea"
+import { Select, SelectContent, SelectItem, SelectTrigger, SelectValue } from "@/components/ui/form/select"
+import { cn } from "@/lib/utils"
+
+// Enhanced Partner interface
+interface Partner {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Partner'. Consider to delete it or rename it.

'Partner' is defined here:


(lint/suspicious/noRedeclare)